exampleSite/config.toml

```diff
@@ -91,18 +91,19 @@ paginate = 10
 
     # Since this template is static, the contact form uses www.formspree.io as a
     # proxy. The form makes a POST request to their servers to send the actual
-    # email. Visitors can send up to a 1000 emails each month for free.
+    # email. Visitors can send up to a 50 emails each month for free.
     #
     # What you need to do for the setup?
     #
-    # - set your email address under 'email' below
+    # - register your account to https://formspree.io/register
+    # - login and create new form
+    # - set your form's endpoint under 'endpoint' below (without https://formspree.io/ from https://formspree.io/ENDPOINT )
     # - upload the generated site to your server
-    # - send a dummy email yourself to confirm your account
-    # - click the confirm link in the email from www.formspree.io
+    # - test a dummy email yourself
     # - you're done. Happy mailing!
     #
-    # Enable the contact form by entering your Formspree.io email
-    email = "your@email.com"
+    # Enable the contact form by entering your Formspree.io endpoint
+    endpoint = "sample/of/endpoint"
```

Comment:
can you rename the variable to `action` ?

Reply:
Of course, OK.
I'll change a variable name and comment.